iree/hal/vulkan/timepoint_util.h

```diff
@@ -153,7 +153,8 @@ class TimePointFencePool final : public RefObject<TimePointFencePool> {
 
   absl::Mutex mutex_;
 
-  IntrusiveList<TimePointFence> free_fences_ ABSL_GUARDED_BY(mutex_);
+  IntrusiveList<std::unique_ptr<TimePointFence>> free_fences_
```

Comment:
"We use unique_ptr to maintain ownership here given that IntrusiveList does no memory management."

Reply:
Reworded a bit to keep it on one line :P